.github/workflows/check-labels.yml

```diff
@@ -11,11 +11,13 @@ on:
     branches: [main]
     paths-ignore: [.github]
```

Comment:
We probably don't need this for consistency.
```suggestion
```

Reply:
Removed after discussion.